src/components/pages/document.yml

```diff
@@ -80,6 +80,9 @@ document:
   archiveAlert:
     en: This document is archived and accessible only by users with admin or maintainer role.
     de: Dieses Dokument ist archiviert und nur für Benutzer mit der Rolle Admin oder Maintainer zugänglich.
+  archiveAlertRedirectMarkdown:
+    en: The other users are automatically redirected to [this document]({documentUrl}).
+    de: Die andere Benutzer sind zu [dieses Dokument]({documentUrl}) weitergeleitet.
```

Comment:
Die andere**n** Benutzer werden automatisch zu \[diese**m** Dokument]({documentUrl}) weitergeleitet.

Reply:
Thank you for the highlighting 😮‍💨 Corrected!